tests/test_meshtastic_utils_skew_drain_coverage.py

```
@@ -0,0 +1,277 @@
+                    "retries": 1,
+                }
+            }
+            result = connect_meshtastic(passed_config=config)
+
+        assert result is None
+        assert mu._relay_startup_drain_deadline_monotonic_secs is None
+        assert mu._startup_packet_drain_applied is False
+        first_client.close.assert_called_once()
+
+    def test_timeout_breaks_on_shutdown(self):
+        def _timeout_then_shutdown(*_args, **_kwargs):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_timeout_then_shutdown`

Add return type annotation: `NoReturn`

(ANN202)